src/Knp/Component/Pager/Paginator.php

```diff
@@ -104,22 +104,33 @@ public function paginate($target, $page = 1, $limit = 10, array $options = array
             }
         }
         
+        // Fix symfony 4.3 deprecation
+        $newDispatch = method_exists(EventDispatcher::class, 'callListeners');
```

Comment:
You can't use `::class`, since this library is supporting php>=5.3

Reply:
As a possible workaround, you can target branch 2.0 instead of master. In 2.0, minimum php version is 7.1

Reply:
> You can't use ::class, since this library is supporting php>=5.3

Yep. but it is only way to check private method exists as I find.

Reply:
> you can target branch 2.0 instead of master

Is not `master` should contain the latest changes ?

Reply:
Not necessarily. In this case, master is aligned with current major release